Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour

```

Comment:
its good to have a character container that contains movement, animation, interactions, inventory and others, we will do it in another branch later when the features come together